src/style.css

```diff
@@ -1,3 +1,87 @@
+:root {
+  --active-link: #00acdc;
+  --text-color: #000;
+  --bg-color: #fff;
+  --shadow-color: 0px 2px 4px 0px #0000000d;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: var(--bg-color);
+  box-shadow: 0 2px 4px 0 var(--shadow-color);
+}
+
+.nav {
+  display: flex;
+  align-items: center;
+  height: 60px;
+  margin-right: 50px;
+}
+
+.logo-link {
+  display: flex;
+  align-items: center;
+  margin-left: 50px;
+}
+
+.lists {
+  display: flex;
+  list-style: none;
+  padding: 0;
+  height: 100%;
+}
+
+.list {
+  position: relative;
+  height: 100%;
+  margin: 0 10px;
```

Comment:
Be consistent with your vertical margins. You have a margin of 0 10px here, which includes horizontal margins. Ensure that you are not setting both top and bottom margins throughout your CSS to maintain consistency.